src/components/policy/test/policy_manager_impl_test.cc

```diff
@@ -104,7 +106,7 @@ class PolicyManagerImplTest : public ::testing::Test {
   NiceMock<MockPolicyListener> listener;
 
   void SetUp() OVERRIDE {
-    manager = new PolicyManagerImpl();
+    manager = new PolicyManagerImpl("testing", 1, 10000);
```

Comment:
@OHerasym Magic numbers? Why 10000 and not 20000? May be it's worth to use some named constants?